help/search-for-messages.md

```diff
@@ -96,6 +96,7 @@ Zulip offers the following filters based on the location of the message.
 * `has:attachment`: Search messages that contain an [uploaded
   file](/help/share-and-upload-files).
 * `has:image`: Search messages that contain uploaded or linked images or videos.
+* `has:reaction`: Search messages with [reactions](/help/emoji-reactions).
```

Comment:
I think probably spelling out "emoji reactions" in the description here is appropriate.